src/components/block/FooterCTACard/FooterCTACard.stories.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import FooterCTACard from "./FooterCTACard";
+
+export default {
+  title: "Block/FooterCTACard",
+  component: FooterCTACard,
+};
+
+const Template = ({ title, body, buttonText }) => (
+  <div style={{ backgroundColor: "black", width: "480px" }}>
+    <FooterCTACard title={title} body={body} buttonText={buttonText} />
+  </div>
+);
+export const DefaultFooterCTACard = Template.bind({});
+DefaultFooterCTACard.args = {
+  title: "Get Involved",
+  body: "Volunteer with us and put your professional skills to good use.",
```

Comment:
This Footer CTA Card `body` property serves the same purpose as the Program Card `description` property. I think we should keep the keys consistent throughout all the Card components. Would you prefer 'body' or 'description'?

Reply:
👍 for consistency across components. I have a slight preference for "description", since I feel that "body" has the connotation of a large body of text, but these are pretty small components and the text here is limited to one or two sentence. But I could see this making sense either way.